### PR TITLE
[helm] Add PDB and update label patterns

### PR DESCRIPTION
**Pull request by stevehipwell**
Fixes https://github.com/aws/eks-charts/issues/553

**Description of changes:**

- Add PodDisruptionBudget when using `enableSqsTerminationDraining` with more than 1 replica
- Add `aws-node-termination-handler.selectorLabels` template
- Refactor selector label patterns to use `aws-node-termination-handler.selectorLabels`
- Refactor labels to use the `nindent` idiomatic pattern

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: config/helm/aws-node-termination-handler/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-node-termination-handler
 description: A Helm chart for the AWS Node Termination Handler
-version: 0.15.1
+version: 0.15.2
```
Comment (bwagner5) [inline config/helm/aws-node-termination-handler/Chart.yaml:4]: can you move this back to `0.15.1`? Our release process will automatically increment the chart version to batch any chart updates into a single version update.

Comment (stevehipwell) [reply config/helm/aws-node-termination-handler/Chart.yaml]: Done.

**Review by bwagner5 — APPROVED** [review]
lgtm, thank you!

**Pull request merged**